master/internal/trial_test.go

```diff
@@ -46,9 +46,6 @@ func TestRendezvousInfo(t *testing.T) {
 			{
 				ContainerPort: 1,
 			},
-			{
```

Comment:
`chore: trim unused parts of rendezvous info (#2381)` trimmed a rendezvous port from the response, but since the master never errored on wrong rendezvous (it just dropped them and moved on) this extra rendezvous resulted in incorrect rendezvous info and an error log but no test failure. I altered this behavior (intentionally) so that the trial would fail too, which resulted in me noticing this. i restored the old behavior but also fixed the test.